src/lib/panels.ts

```diff
@@ -21,6 +23,12 @@ export const panelMetadata = {
     icon: PencilIcon,
     component: NoteRecommendationsPanel,
   },
+  'block-comments': {
+    displayName: 'PANEL.comments',
```

Comment:
weird because you assign well the PANEL.comments variable but in english it's written 'Companion' on the web view
![Screenshot from 2024-01-15 21-55-56](https://github.com/noted-eip/web/assets/62654705/22092a0c-b7c1-4e61-bdef-ee38fb084534)